client/my-sites/stats/activity-log-item/activity-actor.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import React, { PureComponent, PropTypes } from 'react';
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import Gravatar from 'components/gravatar';
+
+export default class ActivityActor extends PureComponent {
+	static propTypes = {
+		actor: PropTypes.shape( {
+			display_name: PropTypes.string,
+			login: PropTypes.string,
+			translated_role: PropTypes.string,
+			avatar_url: PropTypes.string,
+		} ),
+	};
+
+	render() {
+		const avatarUrl = get( this.props, [ 'actor', 'avatar_url' ], 'https://www.gravatar.com/avatar/0' );
```

Comment:
Could we save 4 `get` cycles by retrieving `const { actor } = this.props` and passing `actor` as the first parameter of each of these `get()`s?